mailpoet/lib/Automation/Engine/Workflows/Workflow.php

```diff
@@ -30,14 +30,19 @@ class Workflow {
   /** @param Step[] $steps */
   public function __construct(
     string $name,
-    array $steps
+    array $steps,
+    int $id = null
```

Comment:
Did you end up using it somewhere? Maybe I'll see it in the following PRs. I also "almost needed" it but then didn't add it to avoid these simple value objects becoming an ORM. I guess it's OK to add IDs and we may end up doing so anyway, just asking for the reason (that I may discover myself when moving on with the reviews).